assemble/conf/log4j2-service.properties

```diff
@@ -78,6 +78,11 @@ logger.zookeeper.level = error
 logger.accumulo.name = org.apache.accumulo
 logger.accumulo.level = debug
 
+# uncomment if you start seeing Thrift FrameBuffer error messages
+# in the log (GitHub issue #3042)
```

Comment:
> I don't think we need to reference the issue, because it doesn't really help the user get more information at all about this.

Doesn't it provide more context, for example the possibility that it could be something external to the Accumulo cluster that is just connecting to open ports and sending data?

> Also, I'm not sure we need to have these in here, commented out at all. This is one of many many possible trace logs that the user might be interested in, and I don't think we should comment them all out in the example log4j properties file.

In general, I agree with this. The problem here is that the user has to read the source code to find out all the places where trace logging may be available. I guess the solution here is to just change the root log level to `trace` ?

Reply:
I'll revert the commit that changed this file.

Reply:
> In general, I agree with this. The problem here is that the user has to read the source code to find out all the places where trace logging may be available. I guess the solution here is to just change the root log level to trace

That's kinda true of any log statement at any log level. But, I especially don't think this is a burden for trace level. I'm hoping this is going to be something that isn't going to be needed very often once we watch it and figure out what's going on for ourselves.

Reply:
> In general, I agree with this. The problem here is that the user has to read the source code to find out all the places where trace logging may be available.

The following package is the beginning of an attempt to centralize important logging in the source code and make it discoverable by users.  I would like to move more logging to that package.

https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/logging/package-info.java

Reply:
> I would like to move more logging to that package.

You mean as a task for a future release?

Reply:
> You mean as a task for a future release?

yeah, for a future release would be nice to get the log messages better organized and cleaned up.